TFT/src/User/Menu/FeatureSettings.c

```diff
@@ -43,6 +43,9 @@ const LABEL itemMoveSpeed[ITEM_SPEED_NUM] = {
                                             };
 const  u8 item_movespeed[ITEM_SPEED_NUM] = {LABEL_NORMAL_SPEED, LABEL_SLOW_SPEED, LABEL_FAST_SPEED};
 
+#define ACTION_NUM   1
+const LABEL itemAction[ACTION_NUM]       = {LABEL_RESET};
+

```

Comment:
These lines are not required as the Custom Value of the `Reset settings` item does not changes.
```suggestion
```